sos/report/plugins/ceph_mon.py

```diff
@@ -50,7 +53,12 @@ def setup(self):
             "ceph osd metadata",
             "ceph osd erasure-code-profile ls",
             "ceph osd crush show-tunables",
-            "ceph osd crush dump"
+            "ceph osd crush dump",
+            "ceph mgr dump",
+            "ceph mgr metadata",
+            "ceph mgr module ls",
+            "ceph mgr services",
+            "ceph mgr versions"
```

Comment:
This seems very strange to me to move `mgr` commands _out_ of the `ceph_mgr` plugin, even with the note provided. It is something I think users would certainly not expect at the least.

Reply:
@TurboTurtle It's really being taken "out" of ceph-mgr plugun - previously it was all part of the single ceph plugin where each cmd was every unit and and we had all the info w/ sosreports from all units. Now that the plugin has been split, only the relevant commands are run on given nodes. This is all fine.

But the `ceph mgr ...` commands require admin keyring which may not be present on mgr units. If so, we would lose out `ceph mgr ...` data.

I agree that this sounds odd to say `ceph mgr ...` commands may not work on a ceph-mgr node. But this is probably a historic as ceph-mgr was initially part of the ceph-mon daemon and why these commands don't work with just ceph-mgr keyring.

The bottom line is if we kept the `ceph mgr ...` commands in ceph-mgr plugin, these commands may not be a part of *any* sosreport unless ceph-mgr happens to have the admin keyring. So we might miss out those data completely.

Reply:
Ack. In the end, the folks who use this data will, more likely than not, understand the apparent schism here.